source/shared/net/guides/deciding-wallet.rst

```diff
@@ -1,79 +1,100 @@
 
-.. insert link to topic Learn more about the Ledger, when created
+.. Warning::
+   You are solely responsible for keeping your assets secure regardless of which wallet you choose to use. You must never share your private keys, PIN codes, passwords, recovery phrases, Ledgers, or mobile devices with anyone.
+
+What are the requirements for each wallet?
+==========================================
+
+- **Mobile Wallet**: you’ll need an iPhone running iOS 13 or later or an Android phone running Android 8 or later.
+
+- **Desktop Wallet**: you’ll need a computer running Windows, macOS, or Linux, and you’ll need a Ledger Nano S. You’ll also have to run a node, and your computer must meet the hardware requirements for this.
 
 Comparison
 ==========
 
 The following table lists the major features of each wallet.
 
 .. list-table::
-   :widths: 20 20 20
+   :widths: 20 20
    :header-rows: 1
 
-   * - Features
-     - Desktop Wallet
-     - Mobile Wallet
-   * - **Secured by password and biometrics**
-     - Password
-     - Password and biometrics
-   * - **Encrypted**
-     - Yes
-     - Yes
-   * - **Create backup of identities and accounts**
-     - Yes. The backup includes accounts, identities, and the address book.
-     - Yes. The backup includes accounts, identities, address book, and account keys.
-   * - **Storage of private keys**
-     - Stored on the Ledger that is secured by a PIN code and backed up by recovery phrase.
-     - Stored in the wallet.
-   * - **Create and manage single signature accounts**
-     - Yes
-     - Yes
-   * - **Create and manage multi signature accounts**
-     - Yes
-     - Yes
-   * - **Create identities**
-     - Yes
-     - Yes
-   * - **Create and manage baker accounts**
-     - Yes
-     - The Concordium Client can be used to manage baker accounts. Rewards are visible in the wallet.
-   * - **Run a node**
-     - Requirement
-     - Not needed
+   *  - **Desktop Wallet**
+      - **Mobile Wallet**
+   *  - Secured by password
+      - Secured by password and biometrics
+   *  - Encrypted
+      - Encrypted
+   *  - Backup includes accounts, identities, and the address book.
+      - Backup includes accounts, identities, address book, and private keys.
+   *  - Private keys are stored on the Ledger that is secured by a PIN code and backed up by recovery phrase.
+      - Private keys are stored in the wallet.
+   *  - Creation and management of single signature accounts
+      - Creation and management of single signature accounts
+   *  - Creation and management of multi signature accounts
+      - No creation and management of multi signature accounts
```

Comment:
I think maybe this should just be NA. It is easy to miss the No in the beginning.